lib/asciidoctor/converter/semantic-html5.rb

```diff
@@ -53,10 +69,52 @@ def generate_document_title node
     end
   end
 
+  def generate_authors node
+    return if node.authors.empty?
+
+    if node.authors.length == 1
+      %(<p class="byline">
+#{format_author node, node.authors.first, 1}
+</p>)
+    else
+      result = ['<ul class="byline">']
+      node.authors.each_with_index do |author, index|
+        result << "<li>#{format_author node, author, index + 1}</li>"
+      end
+      result << '</ul>'
+      result.join LF
+    end
+  end
+
+  def format_author node, author, index
+    with_context 'author' do
+      %(<span class="author" data-index="#{index}">#{node.sub_replacements author.name}#{author.email ? %( #{node.sub_macros author.email}) : ''}</span>)
```

Comment:
Not sure if the `data-index` is really useful... I guess it's relatively to compute the index in JavaScript and in CSS we can use [:nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) to select an element based on its position in a group of siblings.

Reply:
I agree. This seems redundant.